cfgov/unprocessed/css/organisms/chart.less

```diff
@@ -0,0 +1,46 @@
+/* topdoc
+  name: Chart
+  family: cfgov-organisms
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-chart">
+            <div class="m-chart-image"></div>
+            <div class="m-inset"></div>
+            <p class="m-chart-footnote"></p>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-chart
+            .m-chart-image
+            .m-chart-footnote
+  tags:
+    - cfgov-organisms
+*/
+
+// Import external cfpb-chart-builder stylesheet
+@import (less) 'cfpb-chart-builder.css';
+
+.o-chart {
+
+  // Title above charts
+  h3 {
```

Comment:
It looks pretty stupid but I ended up leaving them there because next sprint we'll be styling those elements.